### PR TITLE
Enable external services by default and update documentation

### PR DESCRIPTION
**Pull request by nicksnyder**
part of https://github.com/sourcegraph/sourcegraph/issues/914#issuecomment-440067022

### CODE REVIEW

File: pkg/conf/computed.go [hunk 1]
```diff
@@ -50,8 +50,8 @@ func ExternalServicesEnabled() bool {
 	if ef := Get().ExperimentalFeatures; ef != nil {
 		v = ef.ExternalServices
 	}
-	// default is disabled
-	return v == "enabled"
+	// default is enabled
+	return v != "disabled"
```
Comment (tsenart) [inline pkg/conf/computed.go]: I don't understand this change. Where is the actual default value of `v` set?

Comment (nicksnyder) [reply pkg/conf/computed.go]: This line is exactly where the default is enforced.

Here is a relevant issue:
https://github.com/sourcegraph/sourcegraph/issues/1362

In short, the true default anything in our site config is the zero value for that field (i.e. empty string here). Defaults are enforced in code like this.

File: web/src/site-admin/SiteAdminAddExternalServicePage.tsx [hunk 2]
```diff
@@ -37,7 +38,17 @@ export class SiteAdminAddExternalServicePage extends React.Component<Props, Stat
...
+                const params = new URLSearchParams(this.props.history.location.search)
+                const kind = params.get('kind')
+                if (kind) {
+                    const service = ALL_EXTERNAL_SERVICES.find(s => s.kind === kind.toUpperCase())
+                    if (service) {
+                        return service.kind
+                    }
+                }
+                return GQL.ExternalServiceKind.GITHUB
+            })(),
```
Comment (nicksnyder) [inline web/src/site-admin/SiteAdminAddExternalServicePage.tsx]: @felixfbecker I am going to merge this PR soon, but I would appreciate your review on this particular snippet. I need this component to respond to a url query parameter. This works, but I don't know if there is a more idiomatic way to do this.

Comment (felixfbecker) [reply web/src/site-admin/SiteAdminAddExternalServicePage.tsx]: The code LGTM, but this only gets evaluated on component instantiation and not when `this.props.history.location.search` changes. I don't see `kind` being changed anywhere, so I would recommend to remove this state field and call this as a function.

**Pull request merged**